tests/framework/utils/k8s_helper.go

```diff
@@ -496,16 +497,21 @@ func (k8sh *K8sHelper) WaitForLabeledPodsToRunWithRetries(label string, namespac
 	return fmt.Errorf("Giving up waiting for pod with label %s in namespace %s to be running", label, namespace)
 }
 
-//WaitUntilPodWithLabelDeleted returns true if a Pod is deleted within 90s else returns false
+// WaitUntilPodWithLabelDeleted returns true if a Pod is deleted within 90s else returns false
 func (k8sh *K8sHelper) WaitUntilPodWithLabelDeleted(label string, namespace string) bool {
 	options := metav1.ListOptions{LabelSelector: label}
 	for i := 0; i < RetryLoop; i++ {
+		fmt.Printf("DEBUG TEST OUTPUT BEGIN\n")
```

Comment:
do we need these debug prints in the test?

Reply:
Removed them.